src/launch.rs

```diff
@@ -16,9 +16,8 @@ impl TryFrom<Procfile> for Launch {
             launch.processes.push(Process {
                 r#type: ProcessType::from_str(&key)
                     .map_err(ProcfileConversionError::InvalidProcessType)?,
-                command: value,
-                args: Vec::<String>::new(),
-                direct: false,
+                command: vec![String::from("bash")],
+                args: vec![String::from("-c"), format!("exec {value}")],
```

Comment:
Thinking about this more, I'm not sure we need the `exec` here, and in fact I wonder if it would break with procfile entries like `foo && bar`, since the command would end up being `exec foo && bar`, with the exec potentially only applying to `foo`.